src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -4784,6 +4772,8 @@ void UpdateFinalSurfaceHeatBalance(EnergyPlusData &state)
     bool CoolingPanelSysOn;   // true if a simple cooling panel is running
     bool SwimmingPoolOn;      // true if a pool is present (running)
 
+    state.dataHeatBalSurf->QdotRadHVACInPerArea = 0.0;
+    state.dataHeatBalFanSys->QdotRadHVACToPerson = 0.0;
```

Comment:
I have a general concern about the variables being summed in multiple places where before, each specific component was summed in a localized place so we could be sure it was zeroed at the appropriate times. The lack of diffs says you've covered this ok, but I'm trying to follow the code to be sure. Let's take high temp radiant, for example, `UpdateHTRadSourceValAvg` is called here a few lines down.  That function calls `DistributeHTRadGains` which is where `QdotRadHVACInPerArea` get incremented for this type of equipment.  But this is only one of four places that `DistributeHTRadGains` is called.  More comments on this later.

Another thing we're moving away from is using these type of array assignments. It's faster to use a for loop her over surfNum.